content/en/docs/setup/install/multicluster/simplified/index.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Simplified Multicluster Install (experimental)
```

Comment:
For consistency with other experimental tasks, should use square brackets
```suggestion
title: Simplified Multicluster Install [experimental]
```

Reply:
Done.